src/pages/User.js

```diff
@@ -0,0 +1,54 @@
+import { useState, useEffect } from 'react';
+import logo from '../CodeKingTrans.png';
+import Navbar from './Navbar';
+import '../App.css';
+
+function UserCard({ user }){
+    return (
+        <div className='userCard'>
+            <img alt='profile logo' src='https://cdn-icons-png.flaticon.com/512/12225/12225773.png'></img>
+            <h2>{user.firstName + ' ' + user.lastName}</h2>
+        </div>
+    );
+}
+
+
+function User() {
+    const [users, setUsers] = useState([]);
+
+    useEffect(() => {
+        async function getData(){
+            const response = await fetch("https://disc-assignment-5-users-api.onrender.com/api/users")
+            const json = await response.json();
+            setUsers(json);
+        }
+        getData();
+    });
```

Comment:
I fear you have an infinite loop here -- your useEffect will run on every render of your User component. Every time useEffect runs, it will fetch and set the users state again. Because the users state changed, your app will rerender, which runs the useEffect again, which sets the state again, etc etc.
What you want instead is to pass an empty dependency array to useEffect so it only runs on the **initial** render.
```
useEffect( () => {
     ...function body...
}, [])
```